File: src/andre/adventofcode/input/Input.java
package andre.adventofcode.input;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.List;

public class Input {
	public static List<String> readAllLines(String fileURL) {
		
		try {
			return Files.readAllLines(Paths.get(fileURL));
		} catch (IOException e) {
			throw new RuntimeException(e);
		}
	}
	
	public static String readFirstLine(String fileURL) {
		return readAllLines(fileURL).get(0);
	}
}

File: Kotlin/src/Input.kt
import java.io.IOException
import java.nio.file.Files
import java.nio.file.Paths

fun readLinesFromFile(url: String): List<String> {
	try {
		return Files.readAllLines(Paths.get(url)).filterNotNull()
	} catch (e: IOException) {
		System.err.println("File $url not found")
		System.exit(1)
		throw RuntimeException(e)
	}
}

fun readStringFromFile(url: String): String {
	return readLinesFromFile(url).joinToString("")
}